superset/key_value/commands/parameters.py

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing import Dict
+
+from flask_appbuilder.security.sqla.models import User
+from typing_extensions import TypedDict
+
+
+class CommandParameters(TypedDict, total=False):
+    actor: User
+    resource_id: int
+    key: str
+    value: str
+    query_params: Dict[str, str]
```

Comment:
As we don't need to serialize/deserialize these to/from JSON, could we rather use a `@dataclass` here? https://docs.python.org/3/library/dataclasses.html This will make accessing properties more pythonic; `cmd_params.actor` vs `cmd_params["actor"]`.